### PR TITLE
Add a news article for the SR20201 fourth league session

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

File: _posts/2021-04-24-sr2021-final-league-session.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Hills Road Sixth Form College out of the top spot. We saw great perfomances from all
+teams as well as some innovative new strategies, the finals are shaping up to be a
+close call!
+
+Module Ⅳ totally redisigned the arena, while keeping the same game mechanics, meaning teams
+have had to make some large changes to their strategy to ensure continued success. Our new
+larger arena includes more territories, moveable obstacles, and new kinds of territories
+worth varying points.
+
+Now the league stage of the competition is over the Knockouts await.
```
Comment (RealOrangeOne) [inline _posts/2021-04-24-sr2021-final-league-session.md:17]:
```suggestion
Now the league stage of the competition is over, the Knockouts await.
```

File: _posts/2021-04-24-sr2021-final-league-session.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+There will be one play-in match between the bottom two teams in order to reduce
+them to one team followed by an ordinary single-elimination knockout of the 16
+remaining teams. Pairings are arranged such that the higher the position in the league
```
Comment (RealOrangeOne) [inline _posts/2021-04-24-sr2021-final-league-session.md:20]:
```suggestion
remaining teams. Pairings are arranged such that the higher your position in the league
```

File: _posts/2021-04-24-sr2021-final-league-session.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+the easier your path to the final.
```
Comment (PeterJCLaw) [inline _posts/2021-04-24-sr2021-final-league-session.md:21]: or maybe? Think it's fine as-is though.
```suggestion
remaining teams. Pairings are arranged such that the higher a team's position in the league
the easier their path to the final.
```

File: _posts/2021-04-24-sr2021-final-league-session.md (hunk 1 above)
Comment (PeterJCLaw) [reply _posts/2021-04-24-sr2021-final-league-session.md:17]: Don't think this is needed?

File: _posts/2021-04-24-sr2021-final-league-session.md (hunk 3 above)
Comment (sedders123) [reply _posts/2021-04-24-sr2021-final-league-session.md]: 4d48374

File: _posts/2021-04-24-sr2021-final-league-session.md (hunk 1 above)
Comment (sedders123) [reply _posts/2021-04-24-sr2021-final-league-session.md:18]: Yeah not sure this is needed

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**